include/RAJA/pattern/params/kernel_name.hpp

```diff
@@ -5,24 +5,21 @@
 
 namespace RAJA
 {
-namespace expt
-{
 namespace detail
 {
 
-struct KernelName : public ForallParamBase
+struct Name : public ::RAJA::expt::detail::ForallParamBase
```

Comment:
Why is `ForallParamBase` in `expt` namespace? It seems `detail` would be sufficient.